.ansible-lint

```diff
@@ -38,11 +38,13 @@ mock_roles:
   - ceph-restapi
   - ceph-rgw
   - ceph-validate
+  - ensure-docker
 mock_modules:
   - ceph_volume
 skip_list:
-  - parser-error  # AnsibleParserError.
   - fqcn-builtins
+  - parser-error  # AnsibleParserError.
+  - schema
```

Comment:
This is a rather generic skip, are you sure we need it?

Reply:
Laziness. I look again.